symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/datafeed/impl/InMemoryDatafeedIdRepository.java

```diff
@@ -0,0 +1,58 @@
+package com.symphony.bdk.core.service.datafeed.impl;
+
+import com.symphony.bdk.core.config.model.BdkConfig;
+import com.symphony.bdk.core.service.datafeed.DatafeedIdRepository;
+import java.util.Optional;
+import lombok.extern.slf4j.Slf4j;
+import org.apiguardian.api.API;
+
+/**
+ * The implementation of {@link DatafeedIdRepository} interface for a
+ * non-persistent in memory Datafeed.
+ */
+@Slf4j
+@API(status = API.Status.INTERNAL)
+public class InMemoryDatafeedIdRepository implements DatafeedIdRepository {
```

Comment:
Could you then remove the one from [test sources](https://github.com/finos/symphony-bdk-java/blob/main/symphony-bdk-core/src/test/java/com/symphony/bdk/core/test/InMemoryDatafeedIdRepository.java)? I guess that it will require to update unit tests as well

Reply:
Not sure what you mean here

Reply:
Oh I didn't even notice that was there.